docs/src/developer/reference/config-options.md

```diff
@@ -307,6 +307,19 @@ mls:
 
 This default configuration can be overriden on a per-team basis through the [feature config API](../developer/features.md)
 
+This flag also supports setting an `initialConfig` value, which is applied when a team is created:
+
+```yaml
+# galley.yaml
+mls:
+  initialConfig:
+    protocolToggleUsers: []
+    defaultProtocol: mls
+    supportedProtocols: [proteus, mls] # must contain defaultProtocol
```

Comment:
I'm not sure what the suggestion is here. Do you want to change the name of the field?

Reply:
to avoid the possibility of getting this wrong, i would consider supportedProtocols == defaultProtocol + otherSupportedProtocols, and allow default to be missing from others. this would remove the need for a comment that may easily be overlooked.

but as i said, no strong opinion.

Reply:
I see. I'm not sure it's worth it. It will require an additional type, and more validation (when reading from DB instead of just on patch).